test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Typically `test.sh` doesn't have side effects like installing pkgs and such.  I'm fine if you want to organize code into a script for travis, but let's not call it `test.sh`?